appscale/tools/parse_args.py

```diff
@@ -375,6 +375,32 @@ def add_allowed_flags(self, function):
       self.parser.add_argument('--keyname', '-k',
         default=self.DEFAULT_KEYNAME,
         help="the keypair name to use")
+    elif function == "appscale-show-stats":
+      self.parser.add_argument('--keyname', '-k',
+        default=self.DEFAULT_KEYNAME,
+        help="the keypair name to use")
+      self.parser.add_argument('--show',
+        nargs='+',
+        choices=['nodes', 'processes', 'proxies'],
+        default=['nodes', 'proxies'],
+        help="the list of entities that statistics should be printed")
+      self.parser.add_argument('--roles', '-r',
+        nargs='*',
+        default=[],
+        help="print nodes with specified role(s)")
+      self.parser.add_argument('--order-processes', '-o',
+        nargs=1,
+        choices=['cpu', 'mem', 'name'],
+        default=['cpu'],
+        help="the arguments provide processes statistics order by")
+      self.parser.add_argument('--top',
+        type=int,
+        default=0,
+        help="a number of top-processes to be printed")
```

Comment:
I think `--top` is more suitable, because it chooses the most consumed processes.